src/ring_theory/localization.lean

```diff
@@ -236,6 +236,47 @@ lemma exist_integer_multiples_of_finset (s : finset S) :
   ∃ (b : M), ∀ a ∈ s, is_integer R ((b : R) • a) :=
 exist_integer_multiples M s id
 
+/-- A choice of a common multiple of the denominators of a `finset`-indexed family of fractions. -/
+noncomputable
+def common_multiple {ι : Type*} (s : finset ι) (f : ι → S) : M :=
```

Comment:
I don't really like the word "multiple" here, since it appears in the denominator. How about:
```suggestion
def common_denom {ι : Type*} (s : finset ι) (f : ι → S) : M :=
```